### PR TITLE
add same-case subject

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

**Review by Hamzaelkhatri — CHANGES_REQUESTED** [review]
add this case : 
```console 
$ go run . | cat -e 
$
$ go run A B | cat -e 
1$

**Review by Hamzaelkhatri — APPROVED** [review]

File: subjects/samecase/REDME.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## Same-Case 
+
+### Instructions
+
+Write a program that takes 2 arguments and returns:
+
```
Comment (MSilva95) [inline subjects/samecase/REDME.md:6]: here you need to add the instruction for when you have multiple chars...

`- if you have more than one character per argument do this...`

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: subjects/samecase/REDME.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- If either of the characters is not a letter, return -1
+- If both characters are the same case, return 1
+- If both characters are letters, but not the same case, return 0
+- All outputs should be followed by a new-line character
```
[the remark this answers is not in the document]
Comment (zainabdnaya) [reply subjects/samecase/REDME.md:10]: In Line 10, it has been mentioned or is preferred to use in each sentence, "Followed by a newline."

File: subjects/samecase/REDME.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+### Usage
+
+```console
+$ go run . | cat -e
+$
+$ go run . "p" "w"| cat -e
+1$
+$ go run . "p" "Q" | cat -e
+0$
+$go run . "p" "4"  | cat -e
```
[the remark this answers is not in the document]
Comment (zainabdnaya) [reply subjects/samecase/REDME.md]: fixed

File: subjects/samecase/REDME.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+-1$
+$ go run . "P" "Q" "W" | cat -e
+$
+$go run . A b  | cat -e
```
[the remark this answers is not in the document]
Comment (zainabdnaya) [reply subjects/samecase/REDME.md]: fixed

File: subjects/samecase/REDME.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+-1$
```
[the remark this answers is not in the document]
Comment (zainabdnaya) [reply subjects/samecase/REDME.md]: fixed

File: subjects/samecase/REDME.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MSilva95) [reply subjects/samecase/REDME.md]: Yes i saw what you said in line 10, but i think it looks better like i suggested, you don't need to say (all outputs... character)

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: subjects/zappy/audit/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+### Functional
```
[the remark this answers is not in the document]
Comment (zainabdnaya) [reply subjects/zappy/audit/README.md:1]: I delete it

**Review by MSilva95 — APPROVED** [review]

**Pull request merged**